library/ziti.c

```diff
@@ -329,25 +334,43 @@ extern int NF_accept(nf_connection clt, nf_conn_cb cb, nf_data_cb data_cb) {
     return ziti_accept(clt, cb, data_cb);
 }
 
+static void session_refresh(uv_timer_t *t) {
+    nf_context nf = t->data;
+    struct nf_init_req *req = calloc(1, sizeof(struct nf_init_req));
+    req->nf = nf;
+
+    ZITI_LOG(DEBUG, "refreshing API session");
+    ziti_ctrl_current_api_session(&nf->controller, session_cb, req);
+}
+
 static void session_cb(ziti_session *session, ziti_error *err, void *ctx) {
     struct nf_init_req *init_req = ctx;
+    nf_context nf = init_req->nf;
 
     int errCode = err ? code_to_error(err->code) : ZITI_OK;
 
-    if (init_req != NULL) {
-        if (session) {
-            init_req->nf->session = session;
-            ZITI_LOG(INFO, "logged in successfully => api_session[%s]", session->id);
-        } else {
-            ZITI_LOG(ERROR, "failed to login: %s[%d](%s)", err->code, errCode, err->message);
+    if (session) {
+        ZITI_LOG(DEBUG, "%s successfully => api_session[%s]", nf->session ? "refreshed" : "logged in", session->id);
```

Comment:
maybe info level for this and line 342?     ZITI_LOG(DEBUG, "refreshing API session");

Reply:
I want to cut down on INFO level, anything that's periodic shouldn't fill up the log unless you want to DEBUG